proposals/utils/pdf_diff_logic.py

```diff
@@ -1103,7 +1103,7 @@ def create_context_diff(context, old_proposal, new_proposal):
             DiffSection(WMOSection(old_proposal.wmo), WMOSection(new_proposal.wmo))
         )
 
-        if new_proposal.is_pre_assessment:
+        if not new_proposal.is_pre_assessment:
```

Comment:
Man this nested if is really uninviting. Might be worth tidying up at some point.

Still surprised we didn't catch this though. Wouldn't this have meant that _all_ diffs wouldn't render any trajectories previously?

Reply:
Yes, the nesting is kinda crazy. It works best to look at the logic of the pdf, to make any sense of this. (Also black made this less readable imho ...)

I know! Bit crazy, but yeah, basically it never rendered anything.